src/bloom/command_handler.rs

```diff
@@ -298,15 +302,17 @@ pub fn bloom_filter_reserve(ctx: &Context, input_args: &[ValkeyString]) -> Valke
 
 pub fn bloom_filter_insert(ctx: &Context, input_args: &[ValkeyString]) -> ValkeyResult {
     let argc = input_args.len();
+    let validate_size_limit = ctx.get_flags().contains(ContextFlags::REPLICATED);
     // At the very least, we need: BF.INSERT <key> ITEMS <item>
-    if argc < 4 {
+    if !validate_size_limit && argc < 4 || validate_size_limit && argc < 5 {
```

Comment:
```suggestion
    if argc < 4 {
```

This should handle it already. It is not required to contain tightening ratio on the replicated command - it is optional and we should handle cases where it provided and not provided.

The check above is sufficient